cfgov/jinja2/v1/mmt-cost-of-credit-tool/service-worker.js

```diff
@@ -0,0 +1,73 @@
+importScripts("/static/apps/mmt-cost-of-credit-tool/js/precache-manifest.8e35b59ac761981009ee6cd9b5bb9eaa.js", "https://storage.googleapis.com/workbox-cdn/releases/4.3.1/workbox-sw.js");
```

Comment:
This file is automatically generated by Webpack, and it shouldn't be committed to revision control. You should `git rm` it, and then add `config/jinja2/v1/mmt-cost-of-credit-tool/service-worker.js` to the repo's `.gitignore` file on a new line.

Reply:
I am not sure where this is.  I haven't changed the .gitignore for the whole repo.  I checked in  the mmt-my-money-calendar and I don't see one.